depthai_nodes/ml/messages/composite.py

```diff
@@ -0,0 +1,23 @@
+from typing import Dict, List, Union
+
+import depthai as dai
+
+
+class CompositeMessage(dai.Buffer):
+    """CompositeMessage class for storing composite of (dai.Buffer, float, List) data.
+
+    Attributes
+    ----------
+    _data : Dict[str, Union[dai.Buffer, float, List]]
+        Dictionary of data with keys as string and values as either dai.Buffer, float or List.
+    """
+
+    def __init__(self):
+        super().__init__()
+        self._data: Dict[str, Union[dai.Buffer, float, List]] = {}
+
+    def setData(self, data: Dict[str, Union[dai.Buffer, float, List]]):
```

Comment:
Are type hints strict enough? E.g. we assign Classification msgs inside `create_multi_classification_msg`?

Reply:
I made it dai.Buffer so that we can save multiple forms not just classification messages, but also Keypoints, ImgDetections, XFeat, etc etc. Any of the other major messages